src/Flex.php

```diff
@@ -182,6 +182,27 @@ public function activate(Composer $composer, IOInterface $io)
         }
     }
 
+    private function getLockFilePath()
+    {
+        $composerFile = Factory::getComposerFile();
+        $lockFilePath = "json" === pathinfo($composerFile, PATHINFO_EXTENSION)
+                ? substr($composerFile, 0, -4).'lock'
+                :  $composerFile .  '.lock';
+        $symfonyLock = str_replace("composer.", "symfony.", $lockFilePath);
+        if (getenv("SYMFONY_LOCKFILE_PATH")) {
+            $symfonyLock  = getenv("SYMFONY_LOCKFILE_PATH");
+        }
```

Comment:
IMO if the env var is set you shouldn't even parse `$composerFile` , it saves perfs

Reply:
i am not sure if i get this one a 100%.
computing the "right" lock file based on the `getComposerFile()`  - is needed anyway - for later usage in `isComposerLockMissing` - otherwise - it needs to be computed there, as it was prior to the PR.

maybe i am wrong?